root/multicore/CMakeLists.txt

```diff
@@ -87,18 +87,23 @@ if(ROOT_imt_FOUND)
                     DEPENDS ${GENERATE_EXECUTABLE_TEST})
 endif()
 
-ROOTTEST_GENERATE_EXECUTABLE(tExecutorMT tExecutorMT.cxx LIBRARIES Core Imt Net TreePlayer RIO Hist MathCore)
+if(NOT MSVC)
+   ROOTTEST_GENERATE_EXECUTABLE(tExecutorMP tExecutorMP.cxx LIBRARIES Core MultiProc Imt Net TreePlayer RIO Hist MathCore)
```

Comment:
It is a bit surprising that the MultiProc dependent is not coming transitively from the dependency on Imt ....

Reply:
I think it's the usual problem with roottest built independently: the target that brings dependencies with it would be `ROOT::MultiProc`. Unfortunately that target does not exist when roottest is built as part of ROOT.

The only solution I know of is to list all dependencies explicitly.